core/management/commands/wait_for_db.py

```
@@ -0,0 +1,21 @@
+import time
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    help = "Aguarda o banco de dados ficar disponível."  # noqa: VNE003
+
+    def handle(self, *args, **options):
+        self.stdout.write("Esperando o banco de dados ficar disponível...")
+        attempt = 0
+        while True:
```

The loop at line 13 is `while True`, which creates an infinite retry loop. While this meets the minimal requirement (keep retrying until DB is ready), it may cause the container to hang forever if the DB never becomes reachable. Consider adding a configurable maximum retry count or a timeout (e.g., read from an env var) so the process can fail fast or alert in unrecoverable situations.